### PR TITLE
new version of fluentd does not auto load core libs

### PR DESCRIPTION
**Pull request by jescarri**

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
  tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

**Comment by errm** [comment]
Thanks for the PR . . . I have a few nits but it looks good thanks :)

File: lib/fluent/plugin/in_systemd.rb [hunk 1]
```diff
@@ -31,6 +32,7 @@ def start
     end
 
     def shutdown
+      super
       @running = false
       @thread.join
       pos_writer.shutdown
```
Comment (errm) [inline lib/fluent/plugin/in_systemd.rb:38]: could we call super at the end of shutdown or the pos_writer will never get shutdown.

File: lib/fluent/plugin/in_systemd.rb [hunk 2]
```diff
@@ -42,9 +44,9 @@ def shutdown
 
     def seek
       journal.seek(@pos_writer.cursor || read_from)
-    rescue Systemd::JournalError
-      log.warn("Could not seek to cursor #{@pos_writer.cursor} found in pos file: #{@pos_writer.path}")
-      journal.seek(read_from)
+     rescue Systemd::JournalError
+       log.warn("Could not seek to cursor #{@pos_writer.cursor} found in pos file: #{@pos_writer.path}")
+       journal.seek(read_from)
```
Comment (errm) [inline lib/fluent/plugin/in_systemd.rb:49]: don't think we need this whitespace here.

**Comment by errm** [comment]
Ok Merged

**Pull request closed without merging**